sdk/http-sink/src/main/java/com/google/cloud/trace/http/v1/HttpTraceSink.java

```diff
@@ -0,0 +1,67 @@
+package com.google.cloud.trace.gae.v1;
+
+import com.google.auth.oauth2.OAuth2Credentials;
+import com.google.cloud.trace.v1.sink.TraceSink;
+import com.google.common.io.ByteStreams;
+import com.google.devtools.cloudtrace.v1.Trace;
+import com.google.devtools.cloudtrace.v1.Traces;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.logging.Logger;
+
+public class HttpTraceSink implements TraceSink {
+  private final static Logger logger = Logger.getLogger(HttpTraceSink.class.getName());
+
+  private final OAuth2Credentials oAuth2Credentials;
+  private final String hostUrl;
+
+  public HttpTraceSink(OAuth2Credentials oAuth2Credentials, String hostUrl) {
```

Comment:
Can you add another constructor that defaults hostUrl to https://cloudtrace.googleapis.com?